javascript/javascript1/week1/dog_age.js

```diff
@@ -0,0 +1,13 @@
+let dogYearOfBirth = 2018
+let dogYearFuture = 2027
+let dogYear =dogYearFuture - dogYearOfBirth
+let dogAgeInHumanYear = 16*Math.log(dogYear)+31;
```

Comment:
Just a simple multiplication is needed here. `dogAge=7*humanAge`

Reply:
Thanks Yes you are right I am going to replace it.